packages/databyss-ui/components/Editor/slate/hotKeys.js

```diff
@@ -1,14 +1,25 @@
 import { isKeyHotkey } from 'is-hotkey'
 import { IS_IOS, IS_MAC } from 'slate-dev-environment'
+
+export const IS_LINUX =
```

Comment:
Let's move this to `databyss-ui/lib/dom.js` and import it here and in `cypress/support/commands.js` (rather than it being a parameter on every command).